src/components/EventStream.js

```diff
@@ -124,9 +124,15 @@ const EventStream = ({ debugId }) => {
               const newDate = new Date(startTime);
               const dateAsMs = newDate.getTime();
               const adjustedTimeAsMs = dateAsMs + pingEvent.timestamp;
+
+              let sessionId;
+              if (parsedPing && parsedPing.client_info && parsedPing.client_info.session_id) {
```

Comment:
This should be easy to tweak later to support desktop (possibly) having a different named session id property name.